cdap-common/src/main/java/co/cask/cdap/common/conf/Configuration.java

```diff
@@ -540,6 +540,16 @@ public String getRaw(String name) {
   }
 
   /**
+   * Get all key-value kafka config pairs
+   * @return hashmap of <code>name</code> property and its value
+   */
+  public Map<String, String> getKafkaConfigs() {
```

Comment:
No need to have specialized method for getting kafkaConfigs in `Configuration`, instead from `KafkaServerMain` you can directly call `Configuration.getValByRegex` with appropriate regex.